### ISSUE TITLE
How to setup to train ImageNet data for Object Detection in Yolov8

### ISSUE THREAD

**Issue by TapendraBaduwal** [ultralytics/ultralytics#2277]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

How to  setup to train ImageNet data for Object Detection in Yolov8

**Comment by AyushExel** [comment]
Imagenet is a classification dataset. You can use YOLOv8 classifier to train on it. 
```
yolo classify train data=imagenet 
```

**Comment by FFF-fenghen** [comment]
I have the same question, could anyone show the configurations? like batchsize, lr, epochs, etc, i train Imagenet by default.yaml, but I  fail to get the accuracy as reported.

**Comment by TapendraBaduwal** [comment]
@AyushExel If we train Imagenet  classification dataset using YOLOv8 classifier then can we used yolov8n-cls model for object detection ?

**Comment by AyushExel** [comment]
Imagenet is a classification dataset and yolov8n-cls is a classification model. It can't do object detection

**Comment by TapendraBaduwal** [comment]
@AyushExel if i write below code and add yolov8n.yaml ,coco128.yaml file in one folder and i will give path to below code with dataset folder then this work ?

from ultralytics import YOLO
# Load a model
model = YOLO("yolov8n.yaml")  # build a new model from scratch
model = YOLO("yolov8n.pt")  # load a pretrained model (recommended for training)

# Use the model
results = model.train(data="coco128.yaml", epochs=3)  # train the model
results = model.val()  # evaluate model performance on the validation set
results = model("https://ultralytics.com/images/bus.jpg")  # predict on an image
success = model.export(format="onnx")  # export the model to ONNX format

**Comment by TapendraBaduwal** [comment]
> 
@AyushExel Can you give me a simple way to setup the yolov8n.yaml ,coco128.yaml file with below code or shall i need to clone whole source code to train model on my dataset?

from ultralytics import YOLO

# Load a model
model = YOLO("yolov8n.yaml")  # build a new model from scratch
model = YOLO("yolov8n.pt")  # load a pretrained model (recommended for training)

# Use the model
results = model.train(data="coco128.yaml", epochs=3)  # train the model
results = model.val()  # evaluate model performance on the validation set
results = model("https://ultralytics.com/images/bus.jpg")  # predict on an image
success = model.export(format="onnx")  # export the model to ONNX format

**Comment by TapendraBaduwal** [comment]
@glenn-jocher Dear friend from where i can get yolov8n.yaml file can you give me the link of yolov8n.yaml file ?
